en/hardware/pwm_escs_and_servo.md

```diff
@@ -0,0 +1,132 @@
+  Pixhawk has adopted this common industry pattern and outputs 3.3V levels on recent boards.
+
+Note that on a normal quad rotor motors spin at 100-130 Hz. 
+This means that if the bus operates at 400 Hz the motor receives new commands three to four times during a single revolution, but it is physically impossible to change the blade speed (due to electric effects and blade inertia) that fast.
+
+**Why are people then claiming high update rates are relevant?**
+
+Historically systems used the fast update rates to overcome moving average filters on the speed controller input which were designed to prevent electrical overload due to noise on the reference signal / fast rotation rate changes. 
+This means that the high update rates put the motor controllers into a mode of operation they were not designed for. 
+ESCs designed for multicopters (running e.g. SimonK or AfroESCs) do not have these filters in place and could be just used at 100-250 Hz with the same or better response times.
+Another reason to have a high update rate is when the output of the attitude control loop is not synchronised to the PWM generation. 
+A higher update rate is then desirable to effectively limit the dead time between control output and motor update. 
+In any of these cases the main limiting factor is the PWM signal spec itself, which takes 1-2 ms (0-100% motor speed) to transmit and limits the update rate to < 500 Hz.
+
+## Connecting the Ground Wire
+
+An electric signal is defined as the deviation from a nominal zero-value. 
+For normal signals this reference is ground (GND). 
+Therefore just one wire is never enough to transmit a signal - it always needs a reference (on differential signals, the reference is established by the difference). 
+A wiring setup not in need of a ground reference doesn't exist, and if the path the current has to take to "return" is much longer than the signal wire, complex electromagnetic effects can come into play. 
+The image below shows how noisy the signal becomes if GND is not connected.
+
+![PWM without ground](../../assets/hardware/pwm_esc/pwm_without_gnd.png)
+
+
+## Signal Shape Comparison
+
+As the scope shots below show the Pixhawk signal arrives faster (at 400 Hz when configured for multicopters instead of 50-60 Hz), but the overall signal shape and voltage levels are exactly the same as for common RC receivers.
+
+### FrSky D4R-II 
+
+3.3V levels, 55 Hz (default, users are advised to flash the 27 ms / 37 Hz firmware)
+
+![frsky_d4_rii](../../assets/hardware/pwm_esc/frsky_d4_rii.png)
+
+
+### Futaba R7008SB
+
+3.0V levels, 66 Hz
+
+![futaba_r7008sb](../../assets/hardware/pwm_esc/futaba_r7008sb.png)
+
+### Spektrum AR 10000
+
+3.3V levels, 45 Hz
+
+
+![spektrum_ar_10000](../../assets/hardware/pwm_esc/spektrum_ar_10000.png)
+
+### Pixhawk (PX4FMU v2.4.3)
+
+3.3V levels, 400 Hz (**400 Hz ONLY if configured as multicopter, default is 50 Hz**)
```

Comment:
If you want to mention the 400Hz, we also need to mention OneShot, which runs in sync with the rate controller (currently 250Hz, but should soon be 1kHz).